src/registry.rs

```diff
@@ -90,6 +86,14 @@ pub struct Registry {
     registry: Rc<RefCell<AnyMap>>,
 }
 
+thread_local! {
+    static AREG: Registry = {
```

Comment:
I wonder if it is ok to have global Registry instead of user owned registry?
Not to mention this registry is per thread so there might be unexpected situations when attempting to read global registry

Do you think original `Registry` wasn't suitable as it is?
I think if user would need global registry he could just use `SystemRegistry`

Reply:
Old implement was the same, it uses tls. Also user can not get registry object, only addr of actor via from_registry

Reply:
Oh sorry, ok then